src/test/java/org/symphonyoss/symphony/messageml/elements/form/TableSelectTest.java

```diff
@@ -0,0 +1,1080 @@
+package org.symphonyoss.symphony.messageml.elements.form;
```

Comment:
This tests are way too complicated to understand in my pov. You didn't need to assert all nodes, number of attributes and so forth if the PresentationML and Makrdown are being correctly generated. 

That being said, you should assert just one of each node, and possibly refactor so that you don't have to specify a new String of messageML, expectedPresentationML and expectedMarkdown on each test case.

Finally, I think that for unit test messages whether you don't put it or you put it with an English sentence that defines what happened and why it failed.